contrib/openshift/osba-os-template.yaml

```diff
@@ -0,0 +1,238 @@
+  spec:
+    type: ClusterIP
+    selector:
+      app: osba-open-service-broker-azure
+    ports:
+    - name: http
+      protocol: TCP
+      port: 80
+      targetPort: 8080
+
+- apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    name: osba-redis-pv-claim
+  spec:
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: 10Gi
+
+- apiVersion: extensions/v1beta1
+  kind: Deployment
+  metadata:
+    name: osba-redis
+    labels:
+      app: osba-redis
+      release: "osba"
+  spec:
+    template:
+      metadata:
+        labels:
+          app: osba-redis
+      spec:
+        containers:
+        - name: osba-redis
+          image: "bitnami/redis:4.0.9"
+          imagePullPolicy: "IfNotPresent"
+          env:
+          - name: REDIS_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: osba-redis
+                key: redis-password
+          ports:
+          - name: redis
+            containerPort: 6379
+          livenessProbe:
+            exec:
+              command:
+              - redis-cli
+              - ping
+            initialDelaySeconds: 30
+            timeoutSeconds: 5
+          readinessProbe:
+            exec:
+              command:
+              - redis-cli
+              - ping
+            initialDelaySeconds: 5
+            timeoutSeconds: 1
+          resources:
+            requests:
+              cpu: 100m
+              memory: 256Mi
+          volumeMounts:
+          - name: redis-data
+            mountPath: /bitnami/redis
+        volumes:
+        - name: redis-data
+          persistentVolumeClaim:
+            claimName: osba-redis-pv-claim
+
+- apiVersion: extensions/v1beta1
+  kind: Deployment
+  metadata:
+    name: osba-open-service-broker-azure
+    labels:
+      app: osba-open-service-broker-azure
+      release: "osba"
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app: osba-open-service-broker-azure
+    template:
+      metadata:
+        labels:
+          app: osba-open-service-broker-azure
+          release: "osba"
+      spec:
+        containers:
+          - name: open-service-broker-azure
+            image: microsoft/azure-service-broker:v0.10.0
+            imagePullPolicy: IfNotPresent
+            env:
+            - name: ENVIRONMENT
+              value: ${ENVIRONMENT}
+            - name: AZURE_SUBSCRIPTION_ID
+              value: ${AZURE_SUBSCRIPTION_ID}
+            - name: AZURE_TENANT_ID
+              value: ${AZURE_TENANT_ID}
+            - name: AZURE_CLIENT_ID
+              value: ${AZURE_CLIENT_ID}
+            - name: AZURE_CLIENT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: osba-open-service-broker-azure
+                  key: azure-client-secret
+            - name: AZURE_DEFAULT_LOCATION
+              value: ${AZURE_DEFAULT_LOCATION}
+            - name: REDIS_HOST
+              value: osba-redis
```

Comment:
Users should have the option to use remote Redis, so this should only be `osba-redis` if users have elected an on-cluster Redis. Otherwise, the host (and port) should come from parameters.